artifactory/cli.go

```diff
@@ -1321,6 +1318,7 @@ func nugetLegacyCmd(c *cli.Context) error {
 		SetRepoName(c.Args().Get(1)).
 		SetBuildConfiguration(buildConfiguration).
 		SetSolutionPath(c.String("solution-root")).
+		SetUseNugetV2(c.Bool("nuget-v2-protocol")).
```

Comment:
nuget-v2-protocol --> nuget-v2
Since we have the ```legacyNugetV2``` constant, let's change it there, and use it instead of the string. Actually, this is something we should do for all flags (all commands).
Actually - do we need both ```legacyNugetV2``` and ```nugetV2```?

Reply:
Both are needed because the flags have different settings (the legacy is hidden). The flags map cannot contain both with the same key, hence the different name. I cannot rename it to nuget-v2, do you have a different naming suggestion?